src/main/java/com/mucommander/main/muCommander.java

```diff
@@ -231,15 +234,27 @@ public static void main(String[] args) throws Exception
             return;
         }
 
+        new Thread(() -> {
+            System.out.println("Preloading fonts in JVM...");
+            var pre = System.currentTimeMillis();
+            try {
+                GraphicsEnvironment.getLocalGraphicsEnvironment().getAllFonts();
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+            System.out.println("Preloading fonts completed in: " + (System.currentTimeMillis() - pre) + "ms");
```

Comment:
partially - it address the first part of how much time it takes but it's not clear to me why not to load the fonts in a lazy way by the Theme editor

Reply:
I'm ok with follow-up changes, and I trust you for posting them, but it will be a challenge to track the things we say we'll follow up on.. 
but specifically about this part, does the current code also load all the fonts?

Reply:
@ahadas Using [GraphicsEnvironment](https://docs.oracle.com/javase%2F8%2Fdocs%2Fapi%2F%2F/java/awt/GraphicsEnvironment.html) you cannot load only desired fonts (correct me if I'm wrong), one need to call #getAllFonts() - and this is long process (the longer the more fonts installed in the OS). 

"and I trust you for posting them" - I'm kinda lost here, too - I don't think this requires additional changes.
A suggestion: If you spot that something needs to be changed after merge, then pls create an issue and I will address it?
I remember that I need to remove some log.error entries (or switch them to .debug) - I've marked such places with // TODO. For System.println while initializing OSGi (loading bundles), I would leave them as they are.